extensions/amp-access/0.1/test/test-access-expr.js

```diff
@@ -278,5 +286,8 @@ describe('evaluateAccessExpr', () => {
     expect(() => {
       evaluateAccessExpr('num-a = 10', {'num-a': 10});
     }).to.throw();
+    expect(() => {
+      evaluateAccessExpr('num-1 = 10', {'num-1': 10});
```

Comment:
We don't - the test is for failure. But I wanted have a fail-test here anyway to make sure to not break in the future.